sass/site/primary/_posts-and-pages.scss

```diff
@@ -20,10 +20,43 @@
 	display: inline;
 }
 
+.page,
+.single {
+
+	.entry-header {
+		background: $blue;
+		color: $white;
+		margin-bottom: 95px;
+		margin-left: calc(50% - 50vw);
+		max-width: none;
+		padding: 80px 40px 60px;
```

Comment:
Can you use [the `spacing` mixin](https://github.com/wcus/wcus-2019/blob/master/sass/abstracts/_structure.scss#L10) for this? it sets up a consistent rhythm for margins & paddings. You can use `spacing(4)` to get 90px here.

| spacing(x)  | = px |
|-------------|------|
|    0        |  15  |
| 1 (default) |  20  |
|    2        |  30  |
|    3        |  60  |
|    4        |  90  |

(increases by 30 from here)

Reply:
I also think the spacing needs to be bumped up a bunch - it looks like the header is meant to be the same size regardless of whether it's wrapped to two lines. See the tickets mockup with a screenshot from this PR overlaid:

![Screen Shot 2019-04-23 at 11 27 32 AM](https://user-images.githubusercontent.com/541093/56594622-2b39aa80-65bb-11e9-8c18-9b9617ca1564.png)

Reply:
Do we want to stick with the whole number spacings or fine-tune to try to match the pixel height from Sketch? For example, I measure it at 110px from the top to the "T" in the design. `spacing(4)` gives me 100px and `spacing(5)` gives me 130px accounting for line-height.

Reply:
I see you ended up with `spacing(5)`, that looks better - thanks!